src/views/resources/Resources.tsx

```diff
@@ -0,0 +1,83 @@
+import Head from "next/head";
+import Image from "next/image";
+
+import resources from "assets/data/resources.json";
+
+import aiML from "assets/icons/ai-ml.svg";
+import backend from "assets/icons/backend.svg";
+import database from "assets/icons/database-fill.svg";
+import deploy from "assets/icons/deploy.svg";
+import userInterface from "assets/icons/display.svg";
+import gaming from "assets/icons/gaming.svg";
+import tools from "assets/icons/tools.svg";
+
+import Card from "react-bootstrap/Card";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+
+import Resource from "./Resource/Resource";
+
+import { TitleBanner } from "components";
+
+interface IndexMapValue {
+	svg: any;
+	color: string;
+}
+
+interface IndexMap {
+	[key: string]: IndexMapValue;
+}
+
+const RESOURCE_INDEX_MAP: IndexMap = {
+	"User Interface": { svg: userInterface, color: "gold" } as IndexMapValue,
+	"Backend": { svg: backend, color: "blue" } as IndexMapValue,
+	"Databases": { svg: database, color: "gold" } as IndexMapValue,
+	"Deployment": { svg: deploy, color: "blue" } as IndexMapValue,
+	"Gaming": { svg: gaming, color: "gold" } as IndexMapValue,
+	"AI / Machine Learning": { svg: aiML, color: "blue" } as IndexMapValue,
+	"General Resources": { svg: tools, color: "gold" } as IndexMapValue,
+};
```

Comment:
@RandyHuynh5815, this is a bit more complicated than I was expecting, but it will work for now.